cmd/deploy.go

```diff
@@ -89,19 +90,23 @@ func runDeployCmd(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		var err error
+		var e error
 		switch group.Kind {
 		case config.PackerKind:
 			// Packer groups are enforced to have length 1
-			moduleDir := filepath.Join(groupDir, string(group.Modules[0].ID))
-			err = deployPackerGroup(moduleDir)
+			subPath, err := modulewriter.DeploymentSource(group.Modules[0])
```

Comment:
nit. This line may create backward compatibility issues in the future (old deployment folder, new ghpc with changed `DeploymentSource` logic). I understand that `filepath.Join(groupDir, string(group.Modules[0].ID))` has exactly the same problem. We may want to explore persisting the mapping `module -> dir` somewhere.

Reply:
Would your concern go away if we decided to move Packer modules "up" one level in the directory structure? Probably by eliminating the module ID from the path? We are already enforcing Packer groups to be 1-module long anyhow.

Reply:
I am going to close this conversation as PR is approved but we can continue it offline.